scrapers/WillTileXXX.yml

```
@@ -2,7 +2,7 @@ name: WillTileXXX
 sceneByURL:
   - action: scrapeXPath
     url:
-      - https://www.willtilexxx.com/updates
+      - https://willtilexxx.com/updates
```

```suggestion
      - willtilexxx.com/updates
```
Don't need to specify protocol